Dockerfile

```diff
@@ -7,7 +7,8 @@ COPY ./requirements.txt /requirements.txt
 COPY ./entrypoint.sh /bin/entrypoint.sh
 
 RUN yum install -y epel-release &&\
-    yum install -y gcc git python34-pip python34-requests httpd httpd-devel python34-devel &&\
+    yum install -y openssl-devel &&\
+    yum install -y gcc git python36-pip python36-requests httpd httpd-devel python36-devel &&\
```

Comment:
Have you included this in the RHEL dockerfile upstream as well?

Reply:
That is in the base image itself. https://github.com/fabric8-analytics/fabric8-analytics-npm-insights/blob/master/Dockerfile.rhel